app/src/org/commcare/views/widgets/RecordingFragment.java

```diff
@@ -433,10 +438,20 @@ public void onRecordingConfigChanged(List<AudioRecordingConfiguration> configs)
                 if (hasRecordingGoneSilent(configs)) {
                     if (!inPausedState) {
                         pauseRecording();
+                        NotificationUtil.showNotification(
+                                getContext(),
+                                CommCareNoficationManager.NOTIFICATION_CHANNEL_USER_SESSION_ID,
+                                RECORDING_NOTIFICATION_ID,
+                                Localization.get("recording.paused.due.another.app.recording.title"),
+                                Localization.get("recording.paused.due.another.app.recording.message"),
+                                new Intent(getContext(), DispatchActivity.class)

```

Comment:
Does on clicking notification, user resumes the recording screen or get taken out of form ?